docs/docs/reference/singleton-types.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: doc-page
+title: "Literal Singleton Types"
+---
+
+Singleton types is one of the most popular features in Scala. Literal Singleton types takes it a step further.
+Right now, a popular way to replicate literal singleton types is to create macros and/or use libraries such as
+`shapeless` due to the lack of a syntax to express them.
+
+With Literal Singelton types, it is possible to do something like
+
+```scala
+val t: "Jedi".type = "Jedi"
+```
+or
+```scala
+val t: 42.type = 42
+```
+or
+```scala
+val t: 42 = 42
+```
+
+to use literal Singleton types. Trying to do the same in Scala produces the following error:
```

Comment:
I think `to use literal Singleton types` is redundant with `With Literal Singelton types`. What do you think ?